src/System.Configuration.ConfigurationManager/src/MatchingRefApiCompatBaseline.netstandard.txt

```diff
@@ -0,0 +1,12 @@
+Compat issues with assembly System.Configuration.ConfigurationManager:
+MembersMustExist : Member 'System.Configuration.ConfigurationSettings..ctor()' does not exist in the implementation but it does exist in the contract.
```

Comment:
@JeremyKuhne can you please take a look at these to see if they should be in the ref?

Reply:
ok